src/components/atom/Atom.tsx

```diff
@@ -2,64 +2,77 @@ import * as React from "react";
 import cn from "classnames";
 import styles from "./Atom.module.scss";
 
+export type AtomColorVariant = "primary" | "white" | "black" | "inherit";
+
+export interface VariantColor {
+  main: string;
+  contrastText: string;
+  dark: string;
+  light: string;
+}
+
 export interface AtomProps {
   size?: number;
   weight?: 8 | 16 | 24 | 32 | 40 | 48;
-  color?: "primary" | "white" | "black" | "inherit";
+  color?: AtomColorVariant;
   spinning?: boolean;
   className?: string;
 }
+export type ColorMap = {
+  [key in AtomColorVariant]: VariantColor;
+};
 
 const defaultWeight = 16;
 
+export const colors: ColorMap = {
```

Comment:
I moved the colors variants within this constant so that it could be shared with the tests. This means that if new colors are added the they will be covered by the existing test.